client/src/sass/_checkout.scss

```diff
@@ -0,0 +1,337 @@
+/*---------------------
+  Checkout
+-----------------------*/
+.coupon__code {
+  color:$color-17;
+  font-size: 14px;
+  border-top: 2px solid$color-28;
+  background: $background;
+  padding: 23px 30px 18px;
+  margin-bottom: 50px;
+
+  span {
+    margin-right: 15px;
+  }
+
+  a {
+    color:$color-17;
+  }
+}
+
+.checkout__title {
+  color: $heading-color;
```

Comment:
One tip:

with `.checkout__title` you could use

`
.checkout {
    &__title {
     ....
    }
}
`

https://www.pullrequest.com/blog/bem-and-sass-whats-the-difference-and-how-to-use-them/

Anyway, it's okay now you don't really need to do that, just for the next time.

Reply:
take note of this tip and keep it as it is